webui/react/src/pages/ModelVersionDetails.tsx

```diff
@@ -0,0 +1,295 @@
+import { CopyOutlined, EditOutlined } from '@ant-design/icons';
```

Comment:
marking that all the same API + try/catch application will need to be applied here as well